tests/unit/test_court_order.py

```diff
@@ -38,6 +38,8 @@ def test_validate_valid_court_order():
     del order_json['effectOfOrder']
     del order_json['orderDate']
     del order_json['orderDetails']
+    del order_json['fileKey']
+    del order_json['fileName']
```

Comment:
See also the UI code to for other properties that should be stored.

Ref: https://github.com/bcgov/business-filings-ui/blob/c7fadad6ca076a2e6497af991891c8e7b9def7ec/src/components/dialogs/AddStaffNotationDialog.vue#L328

Reply:
fileLastModified and fileSize are not expected by API. Is there any new requirement? and the fileName is not required any more as we don't do anything with it (based on the discussion with @thorwolpert)

Reply:
Oh ok. I guess they're still in the filing JSON anyway.